wiki0/src/server/controllers/authtoken.ts

```diff
@@ -17,10 +15,21 @@ controller.get('/', async (req, res) => {
   });
 
   // Tack on hardcoded request body for debug console demonstration purposes
+  const savedSubjectToken = await req.app.locals.redisClient.get(
```

Comment:
also add to the comment above that these values were set by the authorization server

Reply:
please update comment